docs/user_manual/type_data.md

```diff
@@ -47,8 +47,11 @@ Sample categories are used to group sample classes into levels. Sample categorie
 and ghost samples.
 
 {% assign figure = 1 %}
-Sample classes, and the relationships between them, can be customized to suit your requirements. Figure {{figure}}
-shows the sample hierarchy used at OICR.
+Sample classes, and the relationships between them define the sample hierarchy. Both the classes and relationships can
+be customized to suit your requirements. Figure {{figure}} shows the sample hierarchy used at OICR. Sample
+relationships have a parent and child class. This means that if you have a sample of the parent class, you can
+propagate a sample of the child class. In figure {{figure}}, the relationships are illustrated using arrows that point
+from the child class to the parent class.
```

Comment:
`from the child class to the parent class` ? It looks to me like they point from the parent class to the child class...

In the diagram:
* `Single Cell DNA (sotck)` -> `(stock)`
* The arrows from Tissue -> LCM Tube and Slide -> LCM Tube are joined, which could be interpreted as "one tissue plus one slide = one LCM tube"...

Reply:
Fixed. Also, I had removed whole RNA (aliquot) instead of the three subclasses. Fixed that too.